Content.Server/_Starlight/Antags/Abductor/AbductorSystem.cs

```diff
@@ -0,0 +1,164 @@
+        InitializeActions();
+        //InitializeGizmo();
+        //InitializeConsole();
+        //InitializeOrgans();
+        base.Initialize();
+    }
+
+    private void OnAbductorBeaconChosenBuiMsg(Entity<AbductorHumanObservationConsoleComponent> ent, ref AbductorBeaconChosenBuiMsg args)
+    {
+        OnCameraExit(ent.Owner);
+        if (ent.Comp.RemoteEntityProto != null)
+        {
+            var beacon = _entityManager.GetEntity(args.Beacon.NetEnt);
+            var eye = SpawnAtPosition(ent.Comp.RemoteEntityProto, Transform(beacon).Coordinates);
+            ent.Comp.RemoteEntity = GetNetEntity(eye);
+            
+            console = ent.Owner;
+            
+            if (TryComp<HandsComponent>(args.Actor, out var handsComponent))
+            {
+                foreach (var hand in _hands.EnumerateHands(args.Actor, handsComponent))
+                {
+                    if (hand.HeldEntity == null)
+                        continue;
+                    
+                    if (HasComp<UnremoveableComponent>(hand.HeldEntity))
+                        continue;
+
+                    _hands.DoDrop(args.Actor, hand, true, handsComponent);
+                }
+                
+                if (_virtualItem.TrySpawnVirtualItemInHand(ent.Owner, args.Actor, out var virtItem1))
+                {
+                    EnsureComp<UnremoveableComponent>(virtItem1.Value);
+                }
+
+                if (_virtualItem.TrySpawnVirtualItemInHand(ent.Owner, args.Actor, out var virtItem2))
+                {
+                    EnsureComp<UnremoveableComponent>(virtItem2.Value);
+                }
+            }
+
+            var visibility = EnsureComp<VisibilityComponent>(eye);
+
+            Dirty(ent);
+
+            if (TryComp(args.Actor, out EyeComponent? eyeComp))
+            {
+                _eye.SetVisibilityMask(args.Actor, eyeComp.VisibilityMask | (int)VisibilityFlags.Abductor, eyeComp);
+                _eye.SetTarget(args.Actor, eye, eyeComp);
+                _eye.SetDrawFov(args.Actor, false);
+
+                if (!HasComp<StationAiOverlayComponent>(args.Actor))
+                    AddComp(args.Actor, new StationAiOverlayComponent { AllowCrossGrid = true , Cosmetic = true});
+                if (!TryComp(eye, out RemoteEyeSourceContainerComponent? remoteEyeSourceContainerComponent))
+                {
+                    remoteEyeSourceContainerComponent = new RemoteEyeSourceContainerComponent { Actor = args.Actor };
+                    AddComp(eye, remoteEyeSourceContainerComponent);
+                }
+                else
+                    remoteEyeSourceContainerComponent.Actor = args.Actor;
+                Dirty(eye, remoteEyeSourceContainerComponent);
+            }
+
+            AddActions(args);
+
+            _mover.SetRelay(args.Actor, eye);
+        }
+    }
+
+    private void OnCameraExit(EntityUid actor)
```

Comment:
this is the only function that's used for anything that should be included in this pr: the rest can go, and `AbductorSystem.Actions.cs` can get rolled into this file